src/d3-graph.ts

```diff
@@ -225,72 +226,70 @@ export default class D3Graph {
    * @private
    * @return {undefined}
    */
-  _enableClickToCenter() {
-    const that = this
-    this._nodes.on('click', function(d) {
-      const {translation, scale} = that._getGraphTranslationAndScale()
+  private _enableClickToCenter(): void {
+    this._nodes.on('click', (d: any) => {
+      const {translation, scale} = this._getGraphTranslationAndScale()
       const x = translation[0] - d.x
       const y = translation[1] - d.y
-      that._g
+      this._g
         .transition()
-        .duration(that._TRANSITION_DURATION)
+        .duration(D3Graph._TRANSITION_DURATION)
         .call(
-          that._zoomHandler.transform,
+          this._zoomHandler.transform,
           d3.zoomIdentity.translate(x, y).scale(scale)
         )
     })
   }
 
-  _enableKeyboardPanning() {
+  private _enableKeyboardPanning(): void {
     const keymap = {
       LEFT: 37,
       UP: 38,
       RIGHT: 39,
       DOWN: 40,
     }
 
-    const that = this
     d3.select('body')
-      .on('keyup', function() {
-        const {translation} = that._getGraphTranslationAndScale()
+      .on('keyup', () => {
+        const {translation} = this._getGraphTranslationAndScale()
         let offsetRight = 0
         let offsetDown = 0
 
         switch (d3.event.keyCode) {
           case keymap.UP:
-            offsetDown = translation[1] - that._PAN_MOVEMENT_OFFSET
+            offsetDown = translation[1] - D3Graph._PAN_MOVEMENT_OFFSET
             break
           case keymap.DOWN:
-            offsetDown = translation[1] + that._PAN_MOVEMENT_OFFSET
+            offsetDown = translation[1] + D3Graph._PAN_MOVEMENT_OFFSET
             break
           case keymap.LEFT:
-            offsetRight = translation[0] - that._PAN_MOVEMENT_OFFSET
+            offsetRight = translation[0] - D3Graph._PAN_MOVEMENT_OFFSET
             break
           case keymap.RIGHT:
-            offsetRight = translation[0] + that._PAN_MOVEMENT_OFFSET
+            offsetRight = translation[0] + D3Graph._PAN_MOVEMENT_OFFSET
             break
           default:
             break
         }
 
-        that._g
+        this._g
           .transition()
-          .duration(that._TRANSITION_DURATION)
+          .duration(D3Graph._TRANSITION_DURATION)
           .call(
-            that._zoomHandler.transform,
+            this._zoomHandler.transform,
             d3.zoomIdentity.translate(offsetRight, offsetDown)
           )
       })
   }
 
-  _enableNodeHighlightOnHover() {
+  private _enableNodeHighlightOnHover(): void {
```

Comment:
A comment should be added here to describe this function, similar to render() and _renderNodes().